ui/src/components/TextInput.js

```diff
@@ -79,7 +78,7 @@ const TextInputComponent = ({ className, value, disabled, ref = React.createRef(
 
   return (
     <TextInputWrapper disabled={disabled} className={className}>
-      <TextInput {...{ value, disabled, ...props }} componentRef={ref} />
+      <TextInput ref={ref} {...{ value, disabled, ...props }} />
```

Comment:
doing the same here for the ref
```suggestion
      <TextInput {...{ value, disabled, ...props }} ref={ref} />
```

Reply:
ah, never mind... I see now ref is abstracted from the props spread at the Component's function definition. makes no difference in practice. consistency? lol